editoast/src/views/v2/timetable/stdcm.rs

```diff
@@ -468,9 +473,16 @@ async fn build_work_schedules(
     time: DateTime<Utc>,
     maximum_departure_delay: u64,
     maximum_run_time: u64,
+    work_schedule_group_id: Option<i64>,
```

Comment:
If the work_schedule_group provided in the request is `None` we should not retrieve the work schedule.

```suggestion
    work_schedule_group_id: i64,
```

Reply:
I did that to keep the previous behavior (ie to take all work_schedule if nothing is provided).
I just pushed some changes to do what you want, and I had to change `[tests/tests/test_stdcm.py](https://github.com/OpenRailAssociation/osrd/pull/8336/files#diff-ab7effc3c3bbcd58d98213d18d546f1d6c8044a23a25c9b4e6b3d240d6187645)
@flomonster Can you check if it's ok for you ?